src/main/entities/KeywordFinder.java

```diff
@@ -69,32 +69,10 @@ public Key_word findByTitle(String titleK) throws SQLException {
         }
     }
 
-    public List<Key_word> findAllCustom() throws SQLException {
-
-        try (PreparedStatement s = DbContext.getConnection().prepareStatement("SELECT * FROM key_word WHERE prime = FALSE")) {
-
-            try (ResultSet r = s.executeQuery()) {
-
-                List<Key_word> elements = new ArrayList<>();
-
-                while (r.next()) {
-                    Key_word c = new Key_word();
-
-                    c.setKey_word_id(r.getInt("key_word_id"));
-                    c.setTitle(r.getString("title"));
-                    c.setPrime(r.getBoolean("prime"));
-
-                    elements.add(c);
-                }
-                return elements;
-            }
-        }
-    }
-
-    public List<Key_word> findAllPrimary() throws SQLException {
-
-        try (PreparedStatement s = DbContext.getConnection().prepareStatement("SELECT * FROM key_word WHERE prime = TRUE")) {
+    public List<Key_word> findAll(Boolean primeX) throws SQLException {
```

Comment:
Tu bude možno lepšie funkciu findAll nechat bez parametrov - nájde všetky bez ohľadu na to, či je prime alebo nie
a pridať funkciu, ktorá má parameter prime a vyhľadávať prime/not prime KW